.drone.yml

```diff
@@ -2797,7 +2798,8 @@ steps:
   name: grabpl
 - commands:
   - printenv GCP_KEY | base64 -d > /tmp/gcpkey.json
-  - ./bin/grabpl store-packages --edition oss --gcp-key /tmp/gcpkey.json ${DRONE_TAG}
```

Comment:
good to remove this unnecessary passing in of gcp-key. It just complicates the config